backend/tests_configs.ini

```diff
@@ -202,6 +207,7 @@ email.notification.enabled_on_invitation = False
 webdav.ui.enabled = False
 webdav.base_url = https://localhost:3030
 webdav.root_path = /webdav
+debug = True
```

Comment:
does activating debug mode here will not flood travis ? (we do have issue related to too big log file before)

Reply:
I will remove this line, I forgot it after enabling debug to get tracebacks when testing.